### PR TITLE
Add section on which filters are being run to the output files section of pVACseq, pVACbind, and pVACfuse

### LINKED ISSUE #556: Add documentation to the output file section of each command listing filters applied to the filtered file
This can simply list the filters and link back to the filtering commands section for further information.

### PR DESCRIPTION
**Pull request by susannasiebert**
Closes #556

### CODE REVIEW

File: docs/pvacbind/output_files.rst [hunk 1]
```diff
@@ -31,6 +31,20 @@ created):
...
+
+The filtered.tsv file is the all_epitopes file with the following filters
+applied (in order):
+
+- Binding Filter
+- Top Score Filter
+
+Please see the :ref:`Standalone Filter Commands<pvacbind_filter_commands>`
+documentation for more information on each individual filter. The standalone
+filter commends may be useful to reproduce the filtering or to chose different
```
Comment (malachig) [inline docs/pvacbind/output_files.rst:45]: commends -> commands

File: docs/pvacfuse/output_files.rst [hunk 2]
```diff
@@ -34,6 +34,20 @@ created):
...
+
+The filtered.tsv file is the all_epitopes file with the following filters
+applied (in order):
+
+- Binding Filter
+- Top Score Filter
+
+Please see the :ref:`Standalone Filter Commands<pvacfuse_filter_commands>`
+documentation for more information on each individual filter. The standalone
+filter commends may be useful to reproduce the filtering or to chose different
```
Comment (malachig) [inline docs/pvacfuse/output_files.rst:48]: commends -> commands

File: docs/pvacseq/output_files.rst [hunk 3]
```diff
@@ -35,6 +35,22 @@ created):
...
+applied (in order):
+
+- Binding Filter
+- Coverage Filter
+- Transcript Support Level Filter
+- Top Score Filter
+
+Please see the :ref:`Standalone Filter Commands<filter_commands>`
+documentation for more information on each individual filter. The standalone
+filter commends may be useful to reproduce the filtering or to chose different
```
Comment (malachig) [inline docs/pvacseq/output_files.rst:51]: commends -> commands

**Review by malachig — CHANGES_REQUESTED** [review]
This is clear and concise.  Makes sense to me.  Just a couple minor typos noted.

File: docs/pvacbind/output_files.rst (hunk 1 above)
Comment (susannasiebert) [inline docs/pvacbind/output_files.rst:45]:
```suggestion
filter commands may be useful to reproduce the filtering or to chose different
```

File: docs/pvacfuse/output_files.rst (hunk 2 above)
Comment (susannasiebert) [inline docs/pvacfuse/output_files.rst:48]:
```suggestion
filter commands may be useful to reproduce the filtering or to chose different
```

File: docs/pvacseq/output_files.rst (hunk 3 above)
Comment (susannasiebert) [inline docs/pvacseq/output_files.rst:51]:
```suggestion
filter commands may be useful to reproduce the filtering or to chose different
```

**Review by malachig — APPROVED** [review]

**Pull request merged**